account_invoice_triple_discount/tests/test_invoice_triple_discount.py

```diff
@@ -38,6 +38,25 @@ def setUpClass(cls):
         )
         cls.sale_journal = cls.Journal.search([("type", "=", "sale")], limit=1)
 
+    @classmethod
+    def _create_refund(cls, tax=False, date=False, in_type=False):
```

Comment:
`date` and `in_type` are not used in the method, should be removed.

Maybe the whole method might be substituted by calling `init_invoice`? See https://github.com/odoo/odoo/blob/f71626fa6ba413625d7242f5e13d05b8b95be48c/addons/account/tests/common.py#L366